conf/locale/ar/LC_MESSAGES/django.po

```diff
@@ -19516,6 +19516,12 @@ msgstr ""
 msgid "Show Email Task History"
 msgstr "إظهار سجل كافة مهمات البريد الإلكتروني "
 
+#: lms/templates/instructor/instructor_dashboard_2/sensitive_data_download_msg.txt
+msgid ""
+"Contains information protected under the Protection of Privacy Law - the "
+"wrongful transgressor commits an offense"
+msgstr "يحوي معلومات محمية بموجب قانون حماية الخصوصية - المخالف يعاقب"
```

Comment:
I think edX don't accept direct changes to the `.po` files?

So these changes will need to be removed, and the translations pulled from Transifex later by the @edx-transifex-bot .